groovy/gradle/src/org/netbeans/modules/gradle/api/execute/GradleCommandLine.java

```diff
@@ -803,7 +812,34 @@ public void setStackTrace(StackTrace st) {
     }
 
     public void configure(ConfigurableLauncher launcher) {
-        launcher.setJvmArguments(getArgs(EnumSet.of(SYSTEM)));
+        List<String> jvmargs = getArgs(EnumSet.of(SYSTEM));
+        String userHomeDir = System.getProperty("user.home"); 
+        File gradlePropertiesFile = Paths.get(userHomeDir, ".gradle", "gradle.properties").toFile();
+        if (gradlePropertiesFile.exists()){
+            Properties pps = new Properties();
```

Comment:
Probably you would put this as a private method, as actually the priority of the jvm arg resolve is project gradle.properties, root project gradle.properties, then gradle user home gradle. propertties.
Also for the grade user home, please use the GradleSettings.getGradleUserHome()